app/src/components/InputChip/store/easyActionsFactory.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  InputChipAction,
+  InputChipDispatch,
+  InputChipThunkAction,
+} from 'components/InputChip/store/models';
+
+export const createPrimitiveInputChipAction = (
```

Comment:
This looks similar to what the main store uses. Could we define a single abstract function to accomplish this?